packages/ai-bot/main.ts

```diff
@@ -88,27 +133,43 @@ async function sendMessage(
 }
 
 async function sendOption(client: MatrixClient, room: Room, content: string) {
+  log.info(content);
+  let parsedContent = JSON.parse(content);
+  let patch = parsedContent['patch'];
+  if (patch['attributes']) {
+    patch = patch['attributes'];
+  }
+  let id = parsedContent['id'];
+
   let messageObject = {
     body: content,
-    msgtype: 'm.text',
-    formatted_body: content,
+    msgtype: 'org.boxel.command',
+    formatted_body: 'A patch',
     format: 'org.matrix.custom.html',
+    command: {
+      type: 'patch',
+      id: id,
+      patch: {
+        attributes: patch,
+      },
+    },
   };
-  console.log('Sending', messageObject);
+  log.info(JSON.stringify(messageObject, null, 2));
+  log.info('Sending', messageObject);
   return await client.sendEvent(room.roomId, 'm.room.message', messageObject);
```

Comment:
Hmmm I was just going thru this. `sendEvent` assumes the messageObject is of `IContent` 

and `msgtype` doesn't include `org.boxel.*` which to me is confusing  

export enum MsgType {
    Text = "m.text",
    Emote = "m.emote",
    Notice = "m.notice",
    Image = "m.image",
    File = "m.file",
    Audio = "m.audio",
    Location = "m.location",
    Video = "m.video",
    KeyVerificationRequest = "m.key.verification.request",
}

Reply:
Would be nice to make this into an enum. The IContent interface specifies a union type which is MsgType or an arbitrary string. Using a type shared with the boxel side would be good though to avoid mismatched strings causing problems.